docs/modules/rust-guide/examples/mul-deps/deps-main.rs

```diff
@@ -4,7 +4,7 @@ use ic_cdk::export::candid;
 #[import(canister = "multiply_deps")]
 struct CounterCanister;
 
-#[query]
+#[update]
```

Comment:
What's the reason for this change?

Reply:
Akhi clarified that "Inter canister query calls implementation in the replica is a poc and it'll be quite a while before we prioritize turning that into a complete feature".